.github/workflows/clang-format-pull-request.yml

```diff
@@ -0,0 +1,30 @@
+name: clang-format-pull-request
+
+on:
+  pull_request:
+    branches:
+      - release_oneDPL
+      - master
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 1
+
+    - name: Get clang-format
+      run: sudo apt-get install -yqq clang-format-6.0
+
+    - name: Applying clang-format for changed files
+      run: |
+        URL="https://api.github.com/repos/${GITHUB_REPOSITORY}/pulls/${{ github.event.pull_request.number }}/files"
+        FILES=$(curl -s --request GET --get $URL |  jq -r '.[] | .filename')
+        echo $FILES | xargs -n1 -t clang-format --style=file -i
```

Comment:
I checked the presence of `clang-format` command on Ubuntu 18.04 docker image after running `apt-get install clang-format-6.0` and could not find it. Shell we use `clang-format-6.0` instead?